ctms/schemas/bulk.py

```diff
@@ -0,0 +1,54 @@
+import base64
+from datetime import datetime, timezone
+from typing import Literal, Optional, Union
+
+from pydantic import validator
+
+from .base import ComparableBase
+
+BLANK_VALS = [None, "", "null"]  # TODO: Should "null" be allowed?
+
+
+class BulkRequestSchema(ComparableBase):
+    """A Bulk Read Request."""
+
+    start_time: datetime
+
+    end_time: Optional[Union[datetime, Literal[""]]] = None
+
+    @validator("end_time")
+    def end_time_must_not_be_blank(cls, value):  # pylint: disable=E0213
+        if value in BLANK_VALS:
+            return datetime.now(timezone.utc)
+        return value
+
+    limit: Optional[Union[int, Literal[""]]] = 10
```

Comment:
we should add more constraints to the int as well. It should be greater than 0, and have some upper limit like 100 or 1000. I think [conint](https://pydantic-docs.helpmanual.io/usage/types/#constrained-types) will do the heavy lifting. Maybe this?

```suggestion
    limit: Optional[Union[conint(gt=0, le=100), Literal[""]]] = 10
```

I've asked our partner for advice on the upper limit.

Reply:
Our partner regularly uses 100, agrees that 1000 would work as an upper limit

Reply:
Moved default to 100, upper limit to 1000. Pushing in a new commit

Reply:
I think 10 is still a good default, for someone just clicking buttons in the swagger docs 😬